openfecwebapp/constants.py

```diff
@@ -209,11 +209,10 @@
 ])
 
 candidate_status = OrderedDict([
-    ('C', 'Statutory candidate'),
+    ('C', 'For selected election cycle'),
```

Comment:
Sorry, original discussion in this issue: https://github.com/18F/openFEC-web-app/issues/859